os/net/bluetooth/bluetooth.c

```diff
@@ -1325,8 +1461,11 @@ static int start_le_scan(uint8_t scan_type, uint16_t interval, uint16_t window)
 	return 0;
 }
 
-int bt_ble_scan_update(bool fast_scan)
+int bt_le_scan_update_internal(bool fast_scan)
```

Comment:
Please add 'static' keyword.

Reply:
Made the bt_le_scan_update_internal function static